src/proj/lcc.js

```diff
@@ -0,0 +1,11 @@
+goog.module('ngeo.proj.lcc');
```

Comment:
You need goog.module.declareLegacyNamespace() here and in other new goog.module files.